doc/changes/changes48.xml

```diff
@@ -61,11 +61,10 @@ and is now equivalent to <C>function( arg... )</C>, so no changes in the
 existing code are required.
 </Item>
 <Item>
-Introduced <Ref Func="CallWithTimeout" BookName="ref"/> and
-<Ref Func="CallWithTimeoutList" BookName="ref"/> to call a function with
-a limit on the CPU time it can consume. This functionality may not be
-available on all systems and you should check <C>GAPInfo.TimeoutsSupported</C>
-before using this functionality.
+Introduced <C>CallWithTimeout</C> and <C>CallWithTimeoutList</C> to call a
+function with a limit on the CPU time it can consume. This functionality may
+not be available on all systems and you should check
+<C>GAPInfo.TimeoutsSupported</C> before using this functionality.
```

Comment:
Is this sufficient? I imagine there would also be a reference in the future "changes between GAP 4.8 and 4.9" list.